SConstruct

```diff
@@ -586,25 +611,127 @@ def list_sources(base, suffixes):
                     yield os.path.normpath(path)
     return list(_iter(base))
 
-def add_soci_sources(env, object_builder, is_unity):
+def append_sources (result, *filenames, **kwds):
+    result.append([filenames, kwds])
+
+def get_soci_sources(style):
+    result = []
     cpp_path = [
         'src/soci/src/core',
         'src/sqlite',]
-    object_builder.add_source_files(
+    append_sources(result,
```

Comment:
@rec I *want* you to be a stickler. I saw a pep8 style guide checker. I'll see how that works. I'll use this one unless you have a better recommendation:  https://github.com/jcrocholl/pep8